docs/tutorials/tutorial-installing-flox-from-its-repository-on-debian-and-red-hat.mdx

```diff
@@ -0,0 +1,187 @@
+
+```
+sudo apt update && sudo apt install flox -y
+```
+
+
+### Red Hat-based distros
+
+To perform these steps on modern Red Hat-based distributions that use `dnf`, do the following:
+
+First, download Flox’s GPG keyring.
+
+```
+sudo rpm --import https://downloads.flox.dev/by-env/stable/rpm/flox-archive-keyring.asc
+```
+
+Next, add Flox’s repository to ‘/etc/yum.repos.d/flox.repo’.
+
+```
+sudo dnf config-manager --add-repo https://downloads.flox.dev/by-env/stable/rpm
+```
+
+Finally, install Flox.
+
+``` 
+sudo dnf install flox
+```
+
+
+## 2. Installing from Flox's repository  using “yum”
+
+In CentOS 8 and later, as with Fedora, RHEL, Rocky Linux, and other modern Red Hat-based systems, `dnf` is the default package manager. However, CentOS 7 and earlier ship with `yum` as the default, and `dnf` is not available. For obvious reasons, the method described above won’t work.
+
+This section describes an alternative method for installing from Flox’s repository using `rpm` and `yum`. 
+
+Note: even though the method outlined here deals specifically with older versions of CentOS, it _should_ also work on any Red Hat-based distribution for which `yum` is the default.
+
+To install Flox from its repository using `yum`, do the following:
+
+First, download and import Flox’s GPG key.
+
+``` 
+sudo rpm --import https://downloads.flox.dev/by-env/stable/rpm/flox-archive-keyring.asc
+```
+
+Next, configure the Flox repository and install Flox with yum.
+
+```
+sudo yum --repofrompath=flox,https://downloads.flox.dev/by-env/stable/rpm install flox
+```
+
+
+## 3. Removing the Flox repository
+
+To remove Flox’s repository from a Linux system’s sources, do the following.
+
+
+### Debian-based systems
+
+By default, the Flox repository’s source list is in `/etc/apt/sources.list.d/flox.list`. However, you can double-check this by running the following command:
+
+```
+grep -r 'flox' /etc/apt/sources.list /etc/apt/sources.list.d/
+```
+
+Once you’ve confirmed the existence and location of this file, delete it.
+
+```
+sudo rm /etc/apt/sources.list.d/flox.list
+```
+
+Finally, update your package list.
+
+```
+sudo apt update
+```
+
+
+### Red Hat-based systems
+
+By default, the Flox repository’s source list is in `/etc/yum/repos.d/flox.repo`. However, you can double-check this by running the following command:
+
+```
+sudo dnf repolist | grep flox
+```
+
+Delete the `flox.repo` file from `/etc/yum/repos.d/`.
+
+```
+sudo rm /etc/yum/repos.d/flox.repo
+```
+
+Finally, clean the `dnf` cache
+
+```
+sudo dnf clean all
+```
+
+
+[1] On more recent versions of Debian and Ubuntu (since at least Debian Buster and Ubuntu 20.04), it is no longer necessary to run `sudo apt update` after adding a repository via `sudo add-apt-repository`.
+
+
+## 4. Installing from Flox's repository on SUSE/OpenSUSE
+
+This should work, because OpenSUSE/SUSE use yum, but doesn't. The long story is that you can use `rpm` to install the GPG key and then use `yum` to add the repo.
```

Comment:
I think they use `zypper` which isn't yum. Zypper is built in `libresolv` which `dnf` also uses though.

Reply:
If you need assistance on the SuSe stuff, we can pair on it.

Reply:
i thought i remembered zypper using yum to install rpm packages? happy to pair on this!